authors/apps/articles/cloudinary.md

```diff
@@ -0,0 +1,18 @@
+## Introduction
+Cloudinary is the market leader in providing a comprehensive cloud-based image and video management platform. Cloudinary is being used by hundreds of thousands of users around the world, from small startups to large enterprises. We are here to cover your every image and video-related need.
+
+## How it works
+The API uses a third party cloudinary to store user's profile images. Cloudinary also allows the user to update image manipulation presets e.g cropping, scaling and look and feel
+
+## current preset set
+CROP
+    width: 502
```

Comment:
Hmm... Will this crop the pictures uploaded for articles to have the same width and height and so disrupt the original dimensions?

Reply:
We will also handle cropping in the front end so the user will fit their images on the 502 by 502 cropping box

Reply:
Hmm... I think this type of cropping will only be needed on profile images. For article images, we can just scale down the image so if the image was 1500px by 1200px, we want to scale it down to say 500px by 300px. This may require only specifying a width but not height so it retains the dimensions

Reply:
If it's so, shouldn't this markdown be located in profiles application?